pywr/schema/__init__.py

```diff
@@ -153,4 +160,3 @@ def make_component(self, data, **kwargs):
         values = load_parameter_values(model, data)
         print(self, data)
```

Comment:
I am guessing this shouldn't be here!

Reply:
I didn't add this, but happy to remove it.

Reply:
Oh, apologies. Perhaps I misunderstood. Why do you say it shouldn't be there?

Reply:
If it's useful output it should go to the logger, or be removed. I did mean to remove it. I just mentioned it because I spotted it.

Reply:
Print statements removed